doc/api/n-api.md

```diff
@@ -118,6 +118,89 @@ must make use exclusively of N-API by restricting itself to using
 and by checking, for all external libraries that it uses, that the external
 library makes ABI stability guarantees similar to N-API.
 
+## Building
+
+Unlike addons coded in JavaScript, developing and deploying Node native
+addons using N-API requires an additional set of tools. Besides the basic
+tools required to develop for Node, the native addon developer requires a
+toolchain that can compile C and C++ code into a binary. In addition,
+depending upon how the native addon is deployed, the *user* of the native
+addon will also need to have a C/C++ toolchain installed.
+
+For Linux developers, the necessary C/C++ toolchain is typically already
+installed. For systems lacking the necessary toolchain, the
```

Comment:
Maybe it’s just my Ubuntu experience but installing them manually is part of my standard laptop setup routine – I would maybe use wording that implies that packages typically exist, rather than that they are already installed to begin with.

Reply:
Changed.